src/mudblazor/MudBlazor.Template/Program.cs

```diff
@@ -68,7 +69,7 @@
 #endif
 builder.Services.AddDatabaseDeveloperPageExceptionFilter();
 
-builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
+builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = false)
```

Comment:
It seems that you have enabled it back for the non-top-level statement, but didn't change it back to true for the top-level statement.

Reply:
Fixed